packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -48,6 +48,9 @@ typedef SetTextHandler = void Function(String text);
 /// Returned by [SemanticsConfiguration.getActionHandler].
 typedef SemanticsActionHandler = void Function(Object? args);
 
+/// Signature for a function that receives a semantics update and returns no result.
```

Comment:
nit: as per style guide: List at least one example where this typedef is used. E.g.

Used by [SemanticsOwner.onSemanticsUpdate].